src/handlers/webhookTriggers.ts

```diff
@@ -1,12 +1,11 @@
 import { fetchStatuses } from './validators';
```

Comment:
Does this webhook actually work? The webhook I've setup has never been called. Unsure if the Worker is successfully able to make a request.

Reply:
I tried a different syntax/approach to test and apparently it doesn't work. I'm changing it back to how it was when it was working when we tested it.